src/luminol/modules/time_series.py

```diff
@@ -288,7 +290,7 @@ def normalize(self):
         """
         maximum = self.max()
         if maximum:
-            self.values = map(lambda value: value / maximum, self.values)
+            self.values = list(map(lambda value: value / maximum, self.values))
```

Comment:
list comprehension?

... I know this is a matter of preference, but I have a _strong_ preference for list comprehensions over `map` in cases like this.

Reply:
I also prefer using list comprehensions over lambda/map.